### ISSUE TITLE
"Application Error" when deploying to Heroku

### ISSUE THREAD

**Issue by thombehrens** [salesforce-ux/design-system-starter-kit#40]
Hi, can you help me troubleshoot this? I am having issues getting the repo hosted on a heroku app. I'm not making any changes before deploying, but maybe there is some heroku config I am not considering?

Steps to recreate:
1. fork the repository
2. click "deploy to heroku" from the readme

upon navigating to the app, I see:
![image](https://user-images.githubusercontent.com/16739543/80741630-f1618500-8ae7-11ea-83a8-93fcc87ba4e6.png)

logs:
```
2020-04-30T17:33:40.239542+00:00 app[web.1]:           UI: http://localhost:3001
2020-04-30T17:33:40.239930+00:00 app[web.1]:  UI External: http://localhost:3001
2020-04-30T17:33:40.240111+00:00 app[web.1]:  ---------------------------------------
2020-04-30T17:33:40.240453+00:00 app[web.1]: [Browsersync] Serving files from: dist
2020-04-30T17:33:40.326247+00:00 app[web.1]: [Browsersync] Couldn't open browser (if you are using BrowserSync in a headless environment, you might want to set the open option to false)
2020-04-30T17:33:52.416283+00:00 heroku[router]: at=error code=H20 desc="App boot timeout" method=GET path="/" host=tb-deployment-test.herokuapp.com request_id=9ac64566-2fbe-4fae-a908-584c2c168520 fwd="168.149.242.219" dyno= connect= service= status=503 bytes= protocol=https
2020-04-30T17:34:27.911286+00:00 app[web.1]: Error waiting for process to terminate: No child processes
2020-04-30T17:34:28.006932+00:00 heroku[web.1]: State changed from starting to crashed
2020-04-30T17:35:45.107731+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=tb-deployment-test.herokuapp.com request_id=7c8991e3-5283-4bb2-a8d7-c0e116289ed9 fwd="168.149.242.219" dyno= connect= service= status=503 bytes= protocol=https
2020-04-30T17:35:45.453191+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=tb-deployment-test.herokuapp.com request_id=fb920332-ea7c-4ea7-a4dc-76af77682fba fwd="168.149.242.219" dyno= connect= service= status=503 bytes= protocol=https
```

Would love any hints or insights you have! Thanks

**Comment by maxbogdanov** [comment]
Faced same issue. Looks like browserSync is breaking things. Comment out browserSync call at the end of gulpfile.js, that should resolve.

**Comment by Dottenpixel** [comment]
This issue has been fixed in #43, #45, #46, #47, #48, #49

Merge these changes into your forks using the instructions in the README FAQs

Thanks for reporting and let me know if you have any questions.
